source/nodejs/adaptivecards-site/themes/adaptivecards/layout/designer.ejs

```diff
@@ -45,8 +45,54 @@
 	.acd-icon-scopeTemplate::before {
 		content: "\F2B0";
 	}
+
+	@media all and (display-mode: standalone) {
+		.header-holder {
+			display: none;
+		}
+	}
 </style>
 
+<script type="text/javascript">
+
+	const startUrl = window.location.origin + '<%- config.root %>designer/';
+	const iconUrl = window.location.origin + '<%- config.root %>content/icons_blue/blue-512.png';
+
+	const manifest = {
```

Comment:
Can you add comments in code that why you need to generate manifest manually? It will be easy for the next developer to take over on it